site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -8,15 +8,15 @@ extra_permissions_html:
 
 You can use most `chrome.tabs` methods and events without declaring any permissions in the
 extension's [manifest][manifest] file. However, if you require access to the `url`, `pendingUrl`,
-`title`, or `favIconUrl` properties of [`tabs.Tab`][tab], you must declare the `"tabs"` permission
+`title`, or `favIconUrl` properties of [`tabs.Tab`][tab], you must declare the `"host"` permission
 in the manifest, as shown below:
 
 ```json
 {
   "name": "My extension",
   ...
-  "permissions": [
-    "tabs"
+  "host_permissions": [
```

Comment:
Please revert this change. We want to encourage developers to use the least permissions possible in their extensions. In this case, we are only demonstrating how to access the properties that tabs exposes. It would be better to update the intro to this code sample in order to better set the reader’s expectations than to change the code sample itself.